tools/scylla-sstable.cc

```diff
@@ -2856,6 +2856,7 @@ void print_query_results_json(const cql3::result& result) {
             writer.Key(column_metadata[i]->name->text());
             if (!row[i]) {
                 writer.Null();
+                continue;
```

Comment:
Is this not covered in any test?

Reply:
Added unit test in [6bb9907](https://github.com/scylladb/scylladb/commit/6bb990780718a0f8f86fc5386e4b53f0a107881f)